dnsimple/zones_records_test.go

```diff
@@ -90,28 +93,37 @@ func TestZonesService_CreateRecord(t *testing.T) {
 		testMethod(t, r, "POST")
 		testHeaders(t, r)
 
-		want := map[string]interface{}{"name": "foo", "content": "192.168.0.10", "type": "A"}
-		testRequestJSON(t, r, want)
+		// --- FAIL: TestZonesService_CreateRecord (0.00s)
```

Comment:
@jodosha it's a type issue. I was able to reproduce it

```
➜  dnsimple-go git:(zone-record-regions) go test ./...
--- FAIL: TestZonesService_CreateRecord (0.00s)
	dnsimple_test.go:68: Request parameters = map[type:MX name: content:mxa.example.com regions:[SV1 IAD]], want map[content:mxa.example.com type:MX regions:[SV1 IAD]]
FAIL
FAIL	github.com/dnsimple/dnsimple-go/dnsimple	0.072s
ok  	github.com/dnsimple/dnsimple-go/dnsimple/webhook	0.020s
```

changing the [format of the response](https://golang.org/pkg/fmt/) from `%v` to `%#v` revealed the issue

```
--- FAIL: TestZonesService_CreateRecord (0.00s)
	dnsimple_test.go:68: Request parameters = map[string]interface {}{"type":"MX", "name":"", "content":"mxa.example.com", "regions":[]interface {}{"SV1", "IAD"}}, want map[string]interface {}{"content":"mxa.example.com", "type":"MX", "regions":[]string{"SV1", "IAD"}}
FAIL
FAIL	github.com/dnsimple/dnsimple-go/dnsimple	0.064s
ok  	github.com/dnsimple/dnsimple-go/dnsimple/webhook	0.015s
```